fcrepo-http-api/src/test/java/org/fcrepo/http/api/FedoraLdpTest.java

```diff
@@ -839,7 +838,7 @@ public void testPatchObject() throws Exception {
 
 
     @Test
-    @SuppressWarnings({"resource", "unchecked"})
```

Comment:
There is now a warning about a potential resource leak

Reply:
Fixed